dali/operators/math/expressions/expression_impl_gpu.cuh

```diff
@@ -44,8 +51,17 @@ __device__ void ExecuteUnOp(Result *result, const Input *in, int64_t extent) {
 template <ArithmeticOp op, typename Result, typename Left, typename Right>
 __device__ void ExecuteBinOp(Result *result, const Left *l, const Right *r, int64_t extent) {
   using meta = arithm_meta<op, GPUBackend>;
-  for (int64_t i = blockIdx.x * blockDim.x + threadIdx.x; i < extent; i += blockDim.x * gridDim.x) {
-    result[i] = meta::impl(l[i], r[i]);
+  uint32_t start_ofs = (blockDim.x) * blockIdx.x + threadIdx.x;
```

Comment:
in the previous implementation, you used int64_t. Can we use int64_t everywhere?

Reply:
Added by mistake, will revert to previous int64_t change.